Resources/Private/Templates/NewsletterSubscription/Form.html

```diff
@@ -121,7 +121,22 @@
 	</f:if>
 
 	<f:comment>Render button</f:comment>
-	<f:form.submit name="submitType" value="<f:translate key='subscribe' />" class="{settings.form.button.subscribe.class}" />
+	<f:if condition="{settings.form.button.subscribe.useButtonTag}">
+		<f:then>
+			<f:form.button name="submitType" class="{settings.form.button.subscribe.class}" data="{identifier: 'submit'}">
+				<f:translate key='subscribe' />
+				<f:if condition="{settings.form.button.subscribe.enableSpinner}">
+					<span style="display: none;" class="{settings.form.button.subscribe.spinnerClass}" data-identifier="spinner"><!-- --></span>
```

Comment:
Check :) It's already applied within https://github.com/pixelant/pxa_newsletter_subscription/pull/4/commits/6e8679acd8e4d1bf84510193d1900c883df3a8c6